puppet/spec/unit/util/puppetdb/command_spec.rb

```diff
@@ -25,6 +25,7 @@
       it "should issue the HTTP POST and log success" do
         httpok.stubs(:body).returns '{"uuid": "a UUID"}'
         http.expects(:post).with() do | path, payload, headers, options |
+          expect(assert_valid_producer_ts(path)).to be_truthy
```

Comment:
Looking into the rspec tests a little bit more, I realized we are using mocha, so this with block should look like 

```
        http.expects(:post).with() do | path, payload, headers, options |
          param_map = CGI::parse(URI(path).query)
          assert_valid_producer_ts(path) &&
            param_map['certname'].first.should == 'foo.localdomain' &&
            param_map['version'].first.should == '1' &&
            param_map['command'].first.should == 'OPEN_SESAME' &&
            options[:compress] == :gzip &&
            options[:metric_id] == [:puppetdb, :command, 'OPEN_SESAME']
        end
```